src/components/Tag/Tag.scss

```diff
@@ -20,17 +20,65 @@ $icon-size: rem(16px);
     color: var(--p-text-disabled, color('ink', 'lightest'));
   }
 
-  .TagText {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-  }
+  &.clickable {
+    @include unstyled-button;
+    cursor: pointer;
+    padding: 0 spacing(tight);
+    background-color: color('sky');
+
+    // Start - Delete below me during newDesignLanguage rollout
+
+    &:hover,
+    &:focus,
+    &:active {
+      background: color('sky', 'dark');
+    }
+
+    // End - Delete above me during newDesignLanguage rollout
+
+    &:disabled {
+      cursor: default;
+      pointer-events: none;
+    }
 
-  @media (-ms-high-contrast: active) {
-    outline: 1px solid ms-high-contrast-color('text');
+    // stylelint-disable selector-max-specificity
+    // stylelint-disable selector-max-class
+
+    &.newDesignLanguage {
+      background-color: var(--p-action-secondary);
+
+      &:hover {
+        background: var(--p-action-secondary-hovered);
+      }
+
+      @include focus-ring;
+
+      &:focus:not(:active) {
+        @include focus-ring($style: 'focused');
+      }
+
+      &:active {
+        background: var(--p-action-secondary-pressed);
+      }
+
+      &:disabled {
+        background: var(--p-action-secondary-disabled);
+      }
+      // stylelint-enable
+    }
```

Comment:
(left over curly brace typo from my suggestion)

```suggestion
```

Reply:
Hah yea I saw that break the playground earlier and got rid of it 😄